pump/storage/vlog.go

```diff
@@ -480,3 +478,13 @@ func (vlog *valueLog) gcTS(gcTS int64) {
 		logFile.lock.Unlock()
 	}
 }
+
+// ParseFid parses the given file name for the Fid.
+func ParseFid(fName string) (uint32, error) {
+	fName = strings.TrimSuffix(filepath.Base(fName), fileExt)
+	fid64, err := strconv.ParseUint(fName, 10, 32)
```

Comment:
why `fid64`, not `fid32`？

Reply:
Copied from the original implementation.